sdk/spring/azure-spring-cloud-eventhubs-stream-binder/README.md

```diff
@@ -3,6 +3,25 @@
 The project provides **Spring Cloud Stream Binder for Azure Event Hub** which allows you to build message-driven 
 microservice using **Spring Cloud Stream** based on [Azure Event Hub](https://azure.microsoft.com/en-us/services/event-hubs/) service.
```

Comment:
Following the [Link Guidelines](https://github.com/Azure/azure-sdk/blob/master/docs/policies/README-TEMPLATE.md#link-guidelines) we should remove this `en-us` locale.

Reply:
How about we put these links URL at the bottom of the file using reference links?